fmriprep/workflows/bold/base.py

```diff
@@ -993,6 +994,9 @@ def init_func_preproc_wf(bold_file, has_fieldmap=False):
             (bold_hmc_wf, bold_bold_trans_wf, [
                 ("outputnode.xforms", "inputnode.hmc_xforms"),
             ]),
+            (initial_boldref_wf, final_boldref_wf, [
```

Comment:
I thought the idea of having a second go-round was to get a better mask.

Reply:
This could be IMHO okay, but to me, the problem is that we do not know yet what's making the fast-track fall into this while the regular track is fine for the given seed.

We first need to bisect the issue and determine the point where inputs and (or) outputs deviate from the regular track.

We may live with this just fine, but if we do this not knowing why this happened, the regression will be almost certain.

Reply:
the (silent) failure point is [n4_correct](https://github.com/nipreps/niworkflows/blob/e1f4267eb5fd878b2a001f184e1bddbb3f4a6843/niworkflows/func/util.py#L392-L400), though from looking at the inputs it is not very clear why one case is failing while the other is fine. I've compiled the working directories of fast-track(ft)/no fast-track(noft) within a tarball ([ds210_n4_error.tar.gz](https://github.com/nipreps/fmriprep/files/8496251/ds210_n4_error.tar.gz)) if either of you would like to take a look.